pages/registry/cli.mdx

```diff
@@ -1,96 +1,109 @@
 import { Callout } from "nextra-theme-docs";
 
-# `wasmer publish` CLI
-
-`wasmer publish` allows publishing a Wasmer container to the [registry](/registry).
-
-## Usage
-
-### Create a package
-
-Creating a package is very simple, just run:
+# The `package` CLI subcommand
+Wasmer provides you with CLI commands to work with packages, namely the 
+`wasmer package` range of subcommands.
+
+<Callout type="info">
+  **Note:** To publish packages to the [registry](https://wasmer.io), users
+  need an account. When publishing a package, users are prompted to login if
+  they did not already. 
+</Callout>
 
-```bash
-wasmer init
+```
+$ wasmer package help
+Package related commands
+
+Usage: wasmer package [OPTIONS] <COMMAND>
+
+Commands:
+  download  Download a package from the registry
+  build     Build a container from a package manifest
+  tag       Tag an existing package
+  push      Push a package to the registry
+  publish   Publish (push and tag) a package to the registry
+  help      Print this message or the help of the given subcommand(s)
 ```
 
-### Publish a package to Wasmer
+## Publishing a package
+Once a [package manifest](/registry/manifest) is ready, the user can publish
+the package to the registry. In the Wasmer CLI, publishing a package involves
+two steps: _pushing_ the package and _tagging_ it - that is, uploading the
+package to the selected registry and giving it a name and a version the user
+can use to refer to the package. 
 
-Once you set up the values in your [`wasmer.toml` manifest](/registry/manifest), publishing a package to Wasmer is as simple as running the following command:
+Both these steps come with a dedicated CLI subcommand - `wasmer package push`
+and `wasmer package tag`, in order.
 
-```bash
-wasmer publish
+### Pushing a package
+Let's start checking out the help message of the `push` subcommand:
 ```
+$ wasmer package push --help
```

Comment:
I don't think we should make public package push just yet. Let's just mention publish for now